docs/deep/gpu.md

```diff
@@ -100,6 +100,7 @@ The following sections attempt to outline the versions (as best I can figure the
 | v0.4.x | 0.20.0  | 2.7.4      | 11.2 |
 
 > Note: DJL + TensorFlow will currently not work **at all** on Apple silicon (no matter whether you have the Intel or Apple silicon build of QuPath... unless you build TensorFlow from source).
+> TensorFlow also [dropped GPU support on Windows as of version 2.10](https://www.tensorflow.org/install/pip#windows-native), so your best bet is to try to use a previous compatible version or to run QuPath (and TensorFlow) on the [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install).
```

Comment:
> previous compatible version

Of QuPath or TensorFlow?
First option seems bad, second doesn't seem possible... and unless we manage to get it working on the Windows Subsystem for Linux I'd tend towards just stating that we don't have a solution.

Reply:
All very confusing given DJL is bullishly optimistic https://djl.ai/engines/tensorflow/tensorflow-engine/

Reply:
Removed the suggestion of hope

Reply:
Looks like it was dropped in DJL 0.29.0, which moved to v1.0.0 of TensorFlow Java.

Reply:
It does at least greatly simplify our advice for which deep learning platforms to go for.

I'll try to test out WSL tomorrow, although I'll focus on PyTorch since I know that *can* work